streams/stable.json

```diff
@@ -1,92 +1,92 @@
 {
-    "stream": "stable",
+    "stream": "next-devel",
```

Comment:
That looks really weird here. Looks like you mixed stable/next commands to generate that one.

Reply:
Oh yes! Fixing it, Messed up the versions!

Reply:
This is fixed now.